facefusion/face_analyser.py

```diff
@@ -132,15 +132,30 @@ def detect_with_retinaface(temp_frame : Frame, temp_frame_height : int, temp_fra
 	feature_strides = [ 8, 16, 32 ]
 	feature_map_channel = 3
 	anchor_total = 2
-	prepare_frame = numpy.zeros((face_detector_height, face_detector_width, 3))
-	prepare_frame[:temp_frame_height, :temp_frame_width, :] = temp_frame
-	temp_frame = (prepare_frame - 127.5) / 128.0
+	offset_width = (face_detector_width - temp_frame_width) / 2
```

Comment:
you replaced 3 lines with 15 lines of code

extract this to a method and use it for yoloface too - also try to shrink it